package.json

```diff
@@ -11,32 +11,30 @@
     "type": "git",
     "url": "git+https://github.com/comparaonline/ui-grid.git"
   },
-  "author": "",
+  "author": "ComparaOnline <dev@comparaonline.com>",
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/comparaonline/ui-grid/issues"
   },
   "homepage": "https://github.com/comparaonline/ui-grid#readme",
-  "dependencies": {
+  "peerDependencies": {
+    "react": "^0.14.7",
+    "prop-types": "^15.5.10"
+  },
+  "devDependencies": {
     "babel-core": "^6.25.0",
     "babel-loader": "^7.1.1",
     "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "classnames": "^2.2.3",
     "css-loader": "^0.28.4",
+    "cssnano": "^3.10.0",
     "foundation-sites": "^6.4.1",
     "node-sass": "^4.5.3",
     "node-sass-json-importer": "^3.0.2",
     "path": "^0.12.7",
```

Comment:
`path` an exact copy of the NodeJS ’path’ module published to the NPM registry. Prefer the one included on Node.